samples/subsys/fs/fs_sample/boards/ek_ra8d1.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		sdhc1 = &sdhc1;
+	};
+};
+
+&sdhc1 {
+	status = "okay";
```

Comment:
Any reason we can't just put this in the base board devicetree and avoid overlays for every SDHC sample?

Reply:
This is a mistake. We already added this on board devicetree, so this is redundant, I have removed it. Thank you